docs/testplan.md

```diff
@@ -564,24 +564,35 @@ authentication:
 - [ ] Change `second_factor` type to `on` and verify that mfa is required (no option `none` in dropdown)
 
 #### MFA require auth
-Through the CLI, `tsh login` and register a WebAuthn key with `tsh mfa add` (not supported in UI yet).
+Through the CLI, `tsh login` and register a WebAuthn key with `tsh mfa add`.
```

Comment:
we don't need to use the CLI anymore for this so lets rewrite to a general `register a webauthn key`